### PR TITLE
Sm 2021 08 test plan export contacts

### PR DESCRIPTION
**Pull request by sabinammm**

### CODE REVIEW

File: tests/plans/test_export.py [hunk 1]
```diff
@@ -87,11 +91,11 @@ def test_reply_to_mixin(plan_factory, project_factory,
     project_1 = project_factory()
     project_2 = project_factory()
     plan = plan_factory(
-        contact_name='',
+        contact_name='Joe',
```
Comment (fuzzylogic2000) [inline tests/plans/test_export.py:94]: Jane! :)

Comment (fuzzylogic2000) [reply tests/plans/test_export.py:94]: Or actually this is the place to put in your name to leave some marks on the software. :)

**Review by fuzzylogic2000 — APPROVED** [review]
Nice!

File: tests/plans/test_export.py (hunk 1 above)
Comment (sabinammm) [reply tests/plans/test_export.py:94]: Shoot! Next time :)

**Pull request merged**